pkcs11/gkm/test-sexp.c

```diff
@@ -101,6 +133,34 @@ test_parse_key (Test *test, gconstpointer unused)
 	g_assert (ret);
 	g_assert (mpi != NULL);
 	gcry_mpi_release (mpi);
+
+	/* DSA */
```

Comment:
wouldn't it be cleaner to first extend test coverage for the existing functionality and then add new one?